src/visualizers/ModFill.ts

```diff
@@ -28,29 +39,99 @@ modulus to consider.
      **/
     // note will be small enough to fit in a `number` when we need it to.
     modDimension: {
-        default: 10n,
+        default: 150n,
         type: ParamType.BIGINT,
-        displayName: 'Mod dimension',
+        displayName: 'Highest modulus',
         required: true,
         validate: function (n: number, status: ValidationStatus) {
             if (n <= 0) status.addError('Must be positive.')
         },
     },
+    /** md
+- Alpha: The rate at which cells darken with repeated hits.  This
+should be set between 1 (very transparent) and 255 (solid).
```

Comment:
This is called Transparency in the ui. The names should be the same in the ui and the docs.

Reply:
I fixed this throughout (I think)